src/pruneGitHubNotifications.test.ts

```diff
@@ -114,4 +114,27 @@ describe("pruneGitHubNotifications", () => {
 			]
 		`);
 	});
+
+	it("logs a message when no notifications match the filters", async () => {
```

Comment:
[Testing] If we're going with the feature as-is, I'd request a test for the inverse - that when `logFilterWhenEmpty` is false, it doesn't log. I.e.:

```suggestion
	it("does not log a message when no notifications match the filters and logFilterWhenEmpty is false", async () => { ... });
	it("logs a message when no notifications match the filters and logFilterWhenEmpty is true", async () => { ... });
```

...but, TBD, maybe we'll go with something fancier?

Reply:
yea we should probably just do a verbose flag or something! I don't know how you feel about the debug module, but I'm also a big fan of that